openedx/features/course_experience/views/course_reviews.py

```diff
@@ -59,7 +62,8 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         context = {
             'course': course,
             'course_url': course_url,
-            'course_reviews_provider_fragment': course_reviews_provider_fragment
+            'course_reviews_provider_fragment': course_reviews_provider_fragment,
```

Comment:
I wonder if the term `provider` buys us anything here, or if it just makes this harder to read?